src/server/pfs/pfs.go

```diff
@@ -253,6 +263,7 @@ var (
 	fileNotFoundRe            = regexp.MustCompile(`file .+ not found`)
 	outputCommitNotFinishedRe = regexp.MustCompile("output commit .+ not finished")
 	commitNotFinishedRe       = regexp.MustCompile("commit .+ not finished")
```

Comment:
This other one isn't actually used, which is good as it matches both of the others. It'd be nice if there's a way to get rid of these regex entirely after that grpc status change

Reply:
Yeah, I'm going to go ahead and make the base commit not finished regex not matchable by the not finished regex, but we should eventually come up with a better way to handle the types than regex matching.